docs/integrations/aa/candide.md

```diff
@@ -0,0 +1,325 @@
+# Candide
+
+# Recover a Safe Account with Google using AbstractionKit
+
+This is the second of two guides on how to recovery a Safe Account using Google. In the first guide, we show the steps on how to add the recovery method. We recommend starting with the first guide [here](https://docs.candide.dev/wallet/guides/recovery-with-google-using-lit/).
+
+## What is AbstractionKit?
+AbstractionKit is a Typescript Library to easily build on Account Abstraction, with first class support for Safe Accounts. One of their use cases is to enable users to add recovery methods as a backup in case their use their main signer.
+
+To leverage the full potential of Account Abstraction, you can combine Lit with AbstractionKit to enable email / social recovery experience, while using a Smart Account as the smart wallet to sponsor gas for users, batch transactions, and more.
+
+### Relevant Links
+For additional information during this guide:
+
+- [How on-chain guardian recovery works](https://docs.candide.dev/wallet/plugins/recovery-with-guardians/)
+- [Guardian Recovery SDK Reference](https://docs.candide.dev/blog/making-accounts-recoverable/)
+- [Simple Recovery example on GitHub](https://github.com/candidelabs/abstractionkit/tree/experimental/examples/SafeAccountExamples/SocialRecovery)
+- [Lit Documentation Website](https://developer.litprotocol.com/)
```

Comment:
we can prob remove the link to the Lit docs as we're already there

Reply:
Haha true